tests/auth0/handlers/migrations.tests.js

```diff
@@ -0,0 +1,112 @@
+const { expect } = require('chai');
+const migrations = require('../../../src/auth0/handlers/migrations');
```

Comment:
Wouldn't it be better to `import MigrationsHandler from '../../../src/auth0/handlers/migrations'` here? Avoiding the `migrations.default` below.